crates/primitives/src/transaction/pooled.rs

```diff
@@ -220,6 +226,18 @@ impl PooledTransactionsElement {
     }
 }
 
+impl Default for PooledTransactionsElement {
```

Comment:
didn't realize this exists as supertrait of `SignedTransaction`, could we get rid of it? most of those impls doesn't make much sense and only useful in tests

Reply:
yeah, I was thinking the same,

will remove in a followup